Index: cunaguaro-8.0-4/browser/locales/en-US/profile/bookmarks.inc
===================================================================
--- cunaguaro-8.0-4.orig/browser/locales/en-US/profile/bookmarks.inc	2013-04-16 14:15:02.000000000 -0430
+++ cunaguaro-8.0-4/browser/locales/en-US/profile/bookmarks.inc	2013-04-25 16:12:42.680074465 -0430
@@ -7,34 +7,46 @@
 # your locale code, and link to your translated pages as soon as they're 
 # live.
 
-#define bookmarks_title Bookmarks
-#define bookmarks_heading Bookmarks
+#define bookmarks_title Marcadores
+#define bookmarks_heading Marcadores
 
 #define bookmarks_toolbarfolder Bookmarks Toolbar Folder
 #define bookmarks_toolbarfolder_description Add bookmarks to this folder to see them displayed on the Bookmarks Toolbar
 
+# LOCALIZATION NOTE (bookmarks_addons):
+# link title for https://en-US.add-ons.mozilla.com/en-US/firefox/bookmarks/
+#define bookmarks_addons Sitio de Canaima GNU/Linux
+
+# LOCALIZATION NOTE (latest_headlines):
+# link title for the live bookmarks sample, a redirect on
+# http://en-US.fxfeeds.mozilla.com/en-US/firefox/headlines.xml
+# Changing the redirect is subject to approval of l10n-drivers.
+#define latest_headlines Últimas Noticias en Canaima
+#define latest_tweets Últimos Tweets de CanaimaGNULinux
+#define latest_dents Últimos Dents de CanaimaGNULinux
+
 # LOCALIZATION NOTE (getting_started):
 # link title for https://www.mozilla.org/en-US/firefox/central/
-#define getting_started Getting Started
+#define getting_started Participando en Canaima
 
 # LOCALIZATION NOTE (firefox_heading):
 # Firefox links folder name
-#define firefox_heading Mozilla Firefox
+#define firefox_heading Enlaces de Interés
 
 # LOCALIZATION NOTE (firefox_help):
 # link title for https://www.mozilla.org/en-US/firefox/help/
-#define firefox_help Help and Tutorials
+#define firefox_help Soporte
 
 # LOCALIZATION NOTE (firefox_customize):
 # link title for https://www.mozilla.org/en-US/firefox/customize/
-#define firefox_customize Customize Firefox
+#define firefox_customize Reportar un Error
 
 # LOCALIZATION NOTE (firefox_community):
 # link title for https://www.mozilla.org/en-US/contribute/
-#define firefox_community Get Involved
+#define firefox_community Hablar con la Comunidad
 
 # LOCALIZATION NOTE (firefox_about):
 # link title for https://www.mozilla.org/en-US/about/
-#define firefox_about About Us
+#define firefox_about Navegar el Código Fuente
 
 #unfilter emptyLines
Index: cunaguaro-8.0-4/browser/locales/generic/profile/bookmarks.html.in
===================================================================
--- cunaguaro-8.0-4.orig/browser/locales/generic/profile/bookmarks.html.in	2013-04-16 14:15:02.000000000 -0430
+++ cunaguaro-8.0-4/browser/locales/generic/profile/bookmarks.html.in	2013-04-25 16:06:13.092055611 -0430
@@ -10,18 +10,27 @@
 <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=UTF-8">
 <TITLE>@bookmarks_title@</TITLE>
 <H1>@bookmarks_heading@</H1>
-
+    <DT><A HREF="http://canaima.softwarelibre.gob.ve/" ADD_DATE="1223538234" LAST_MODIFIED="1223538246" ICON_URI="http://canaima.softwarelibre.gob.ve/favicon.ico" ICON="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAABY0lEQVQ4jcWSv0rDUBjFf03SJE0VxAcQnJyqFaqDUBCETh1EcPABRCeXgr6DUKSDiEvpC+ggCKYouDk7OneUFkXz/yZx6B9MY1z7Tffe755zz3fugVlXLqvRvO3G47VpmnSvm3/eVbKA9c1VCoqEIyJ21lYAYiBFpEyD97fK6LqOLEnDQ9sGNM5PT7BcHyD+TSJNKygaBgVNZXG+yOXdE53HF1RFHvZ0NTXChGD8OkBekhh8WWhaHgCZGEVWcH3B2fEhtaPGxJ/ECOHI0yCKsGybg2oFXde5uHlABILd6kZKQYLAFyGWbYNhUDQMAHrv/QTA9YNsAsfzAG1kHFzdPwNQr5SQVZUwirI9aOzVck+vbziex8By6PU/EYFABAIUBccXfHy7tNqdxFemcmC5PrIk4Xg+2+slFuZ0fBERhoIgEikFqXSNg1ReXiKOQwC8IKLV7gDpIP0bZdM0J/usKM++fgAl6JNs2MQ/GQAAAABJRU5ErkJggg==" LAST_CHARSET="UTF-8">@bookmarks_addons@</A>
+    <HR>
 <DL><p>
     <DT><H3 PERSONAL_TOOLBAR_FOLDER="true" ID="rdf:#$FvPhC3">@bookmarks_toolbarfolder@</H3>
 <DD>@bookmarks_toolbarfolder_description@
     <DL><p>
-        <DT><A HREF="https://www.mozilla.org/@AB_CD@/firefox/central/" ID="rdf:#$GvPhC3">@getting_started@</A>
+        <DT><A HREF="http://canaima.softwarelibre.gob.ve/unirse" ID="rdf:#$GvPhC3">@getting_started@</A>
+        <DT><A HREF="http://canaima.softwarelibre.gob.ve/news" FEEDURL="http://canaima.softwarelibre.gob.ve/news/aggregator/RSS" ID="rdf:#$HvPhC3">@latest_headlines@</A>
+        <DT><A HREF="http://twitter.com/CanaimaGNULinux" FEEDURL="http://twitter.com/statuses/user_timeline/97067702.rss" ID="rdf:#$HvPhC3">@latest_tweets@</A>
+        <DT><A HREF="http://identi.ca/group/canaimalinux" FEEDURL="http://identi.ca/api/statusnet/groups/timeline/3630.rss" ID="rdf:#$HvPhC3">@latest_dents@</A>
     </DL><p>
+    <HR>
     <DT><H3 ID="rdf:#$ZvPhC3">@firefox_heading@</H3>
     <DL><p>
-        <DT><A HREF="https://www.mozilla.org/@AB_CD@/firefox/help/" ICON="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABGdBTUEAAK/INwWK6QAAABl0RVh0U29mdHdhcmUAQWRvYmUgSW1hZ2VSZWFkeXHJZTwAAAHWSURBVHjaYvz//z8DJQAggJiQOe/fv2fv7Oz8rays/N+VkfG/iYnJfyD/1+rVq7ffu3dPFpsBAAHEAHIBCJ85c8bN2Nj4vwsDw/8zQLwKiO8CcRoQu0DxqlWrdsHUwzBAAIGJmTNnPgYa9j8UqhFElwPxf2MIDeIrKSn9FwSJoRkAEEAM0DD4DzMAyPi/G+QKY4hh5WAXGf8PDQ0FGwJ22d27CjADAAIIrLmjo+MXA9R2kAHvGBA2wwx6B8W7od6CeQcggKCmCEL8bgwxYCbUIGTDVkHDBia+CuotgACCueD3TDQN75D4xmAvCoK9ARMHBzAw0AECiBHkAlC0Mdy7x9ABNA3obAZXIAa6iKEcGlMVQHwWyjYuL2d4v2cPg8vZswx7gHyAAAK7AOif7SAbOqCmn4Ha3AHFsIDtgPq/vLz8P4MSkJ2W9h8ggBjevXvHDo4FQUQg/kdypqCg4H8lUIACnQ/SOBMYI8bAsAJFPcj1AAEEjwVQqLpAbXmH5BJjqI0gi9DTAAgDBBCcAVLkgmQ7yKCZxpCQxqUZhAECCJ4XgMl493ug21ZD+aDAXH0WLM4A9MZPXJkJIIAwTAR5pQMalaCABQUULttBGCCAGCnNzgABBgAMJ5THwGvJLAAAAABJRU5ErkJggg==" ID="rdf:#$22iCK1">@firefox_help@</A>
-        <DT><A HREF="https://www.mozilla.org/@AB_CD@/firefox/customize/" ICON="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABGdBTUEAAK/INwWK6QAAABl0RVh0U29mdHdhcmUAQWRvYmUgSW1hZ2VSZWFkeXHJZTwAAAHWSURBVHjaYvz//z8DJQAggJiQOe/fv2fv7Oz8rays/N+VkfG/iYnJfyD/1+rVq7ffu3dPFpsBAAHEAHIBCJ85c8bN2Nj4vwsDw/8zQLwKiO8CcRoQu0DxqlWrdsHUwzBAAIGJmTNnPgYa9j8UqhFElwPxf2MIDeIrKSn9FwSJoRkAEEAM0DD4DzMAyPi/G+QKY4hh5WAXGf8PDQ0FGwJ22d27CjADAAIIrLmjo+MXA9R2kAHvGBA2wwx6B8W7od6CeQcggKCmCEL8bgwxYCbUIGTDVkHDBia+CuotgACCueD3TDQN75D4xmAvCoK9ARMHBzAw0AECiBHkAlC0Mdy7x9ABNA3obAZXIAa6iKEcGlMVQHwWyjYuL2d4v2cPg8vZswx7gHyAAAK7AOif7SAbOqCmn4Ha3AHFsIDtgPq/vLz8P4MSkJ2W9h8ggBjevXvHDo4FQUQg/kdypqCg4H8lUIACnQ/SOBMYI8bAsAJFPcj1AAEEjwVQqLpAbXmH5BJjqI0gi9DTAAgDBBCcAVLkgmQ7yKCZxpCQxqUZhAECCJ4XgMl493ug21ZD+aDAXH0WLM4A9MZPXJkJIIAwTAR5pQMalaCABQUULttBGCCAGCnNzgABBgAMJ5THwGvJLAAAAABJRU5ErkJggg==" ID="rdf:#$32iCK1">@firefox_customize@</A>
-        <DT><A HREF="https://www.mozilla.org/@AB_CD@/contribute/" ICON="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABGdBTUEAAK/INwWK6QAAABl0RVh0U29mdHdhcmUAQWRvYmUgSW1hZ2VSZWFkeXHJZTwAAAHWSURBVHjaYvz//z8DJQAggJiQOe/fv2fv7Oz8rays/N+VkfG/iYnJfyD/1+rVq7ffu3dPFpsBAAHEAHIBCJ85c8bN2Nj4vwsDw/8zQLwKiO8CcRoQu0DxqlWrdsHUwzBAAIGJmTNnPgYa9j8UqhFElwPxf2MIDeIrKSn9FwSJoRkAEEAM0DD4DzMAyPi/G+QKY4hh5WAXGf8PDQ0FGwJ22d27CjADAAIIrLmjo+MXA9R2kAHvGBA2wwx6B8W7od6CeQcggKCmCEL8bgwxYCbUIGTDVkHDBia+CuotgACCueD3TDQN75D4xmAvCoK9ARMHBzAw0AECiBHkAlC0Mdy7x9ABNA3obAZXIAa6iKEcGlMVQHwWyjYuL2d4v2cPg8vZswx7gHyAAAK7AOif7SAbOqCmn4Ha3AHFsIDtgPq/vLz8P4MSkJ2W9h8ggBjevXvHDo4FQUQg/kdypqCg4H8lUIACnQ/SOBMYI8bAsAJFPcj1AAEEjwVQqLpAbXmH5BJjqI0gi9DTAAgDBBCcAVLkgmQ7yKCZxpCQxqUZhAECCJ4XgMl493ug21ZD+aDAXH0WLM4A9MZPXJkJIIAwTAR5pQMalaCABQUULttBGCCAGCnNzgABBgAMJ5THwGvJLAAAAABJRU5ErkJggg==" ID="rdf:#$42iCK1">@firefox_community@</A>
-        <DT><A HREF="https://www.mozilla.org/@AB_CD@/about/" ICON="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABGdBTUEAAK/INwWK6QAAABl0RVh0U29mdHdhcmUAQWRvYmUgSW1hZ2VSZWFkeXHJZTwAAAHWSURBVHjaYvz//z8DJQAggJiQOe/fv2fv7Oz8rays/N+VkfG/iYnJfyD/1+rVq7ffu3dPFpsBAAHEAHIBCJ85c8bN2Nj4vwsDw/8zQLwKiO8CcRoQu0DxqlWrdsHUwzBAAIGJmTNnPgYa9j8UqhFElwPxf2MIDeIrKSn9FwSJoRkAEEAM0DD4DzMAyPi/G+QKY4hh5WAXGf8PDQ0FGwJ22d27CjADAAIIrLmjo+MXA9R2kAHvGBA2wwx6B8W7od6CeQcggKCmCEL8bgwxYCbUIGTDVkHDBia+CuotgACCueD3TDQN75D4xmAvCoK9ARMHBzAw0AECiBHkAlC0Mdy7x9ABNA3obAZXIAa6iKEcGlMVQHwWyjYuL2d4v2cPg8vZswx7gHyAAAK7AOif7SAbOqCmn4Ha3AHFsIDtgPq/vLz8P4MSkJ2W9h8ggBjevXvHDo4FQUQg/kdypqCg4H8lUIACnQ/SOBMYI8bAsAJFPcj1AAEEjwVQqLpAbXmH5BJjqI0gi9DTAAgDBBCcAVLkgmQ7yKCZxpCQxqUZhAECCJ4XgMl493ug21ZD+aDAXH0WLM4A9MZPXJkJIIAwTAR5pQMalaCABQUULttBGCCAGCnNzgABBgAMJ5THwGvJLAAAAABJRU5ErkJggg==" ID="rdf:#$52iCK1">@firefox_about@</A>
+        <DT><A HREF="http://canaima.softwarelibre.gob.ve/Soporte" ICON_URI="http://canaima.softwarelibre.gob.ve/favicon.ico" ICON="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAABY0lEQVQ4jcWSv0rDUBjFf03SJE0VxAcQnJyqFaqDUBCETh1EcPABRCeXgr6DUKSDiEvpC+ggCKYouDk7OneUFkXz/yZx6B9MY1z7Tffe755zz3fugVlXLqvRvO3G47VpmnSvm3/eVbKA9c1VCoqEIyJ21lYAYiBFpEyD97fK6LqOLEnDQ9sGNM5PT7BcHyD+TSJNKygaBgVNZXG+yOXdE53HF1RFHvZ0NTXChGD8OkBekhh8WWhaHgCZGEVWcH3B2fEhtaPGxJ/ECOHI0yCKsGybg2oFXde5uHlABILd6kZKQYLAFyGWbYNhUDQMAHrv/QTA9YNsAsfzAG1kHFzdPwNQr5SQVZUwirI9aOzVck+vbziex8By6PU/EYFABAIUBccXfHy7tNqdxFemcmC5PrIk4Xg+2+slFuZ0fBERhoIgEikFqXSNg1ReXiKOQwC8IKLV7gDpIP0bZdM0J/usKM++fgAl6JNs2MQ/GQAAAABJRU5ErkJggg==" ID="rdf:#$22iCK1">@firefox_help@</A>
+        <DT><A HREF="http://trac.canaima.softwarelibre.gob.ve/canaima" ICON_URI="http://canaima.softwarelibre.gob.ve/favicon.ico" ICON="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAABY0lEQVQ4jcWSv0rDUBjFf03SJE0VxAcQnJyqFaqDUBCETh1EcPABRCeXgr6DUKSDiEvpC+ggCKYouDk7OneUFkXz/yZx6B9MY1z7Tffe755zz3fugVlXLqvRvO3G47VpmnSvm3/eVbKA9c1VCoqEIyJ21lYAYiBFpEyD97fK6LqOLEnDQ9sGNM5PT7BcHyD+TSJNKygaBgVNZXG+yOXdE53HF1RFHvZ0NTXChGD8OkBekhh8WWhaHgCZGEVWcH3B2fEhtaPGxJ/ECOHI0yCKsGybg2oFXde5uHlABILd6kZKQYLAFyGWbYNhUDQMAHrv/QTA9YNsAsfzAG1kHFzdPwNQr5SQVZUwirI9aOzVck+vbziex8By6PU/EYFABAIUBccXfHy7tNqdxFemcmC5PrIk4Xg+2+slFuZ0fBERhoIgEikFqXSNg1ReXiKOQwC8IKLV7gDpIP0bZdM0J/usKM++fgAl6JNs2MQ/GQAAAABJRU5ErkJggg==" ID="rdf:#$22iCK1">@firefox_customize@</A>
+        <DT><A HREF="http://tribus.canaima.softwarelibre.gob.ve/" ICON_URI="http://tribus.canaima.softwarelibre.gob.ve/favicon.ico" ICON="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAABY0lEQVQ4jcWSv0rDUBjFf03SJE0VxAcQnJyqFaqDUBCETh1EcPABRCeXgr6DUKSDiEvpC+ggCKYouDk7OneUFkXz/yZx6B9MY1z7Tffe755zz3fugVlXLqvRvO3G47VpmnSvm3/eVbKA9c1VCoqEIyJ21lYAYiBFpEyD97fK6LqOLEnDQ9sGNM5PT7BcHyD+TSJNKygaBgVNZXG+yOXdE53HF1RFHvZ0NTXChGD8OkBekhh8WWhaHgCZGEVWcH3B2fEhtaPGxJ/ECOHI0yCKsGybg2oFXde5uHlABILd6kZKQYLAFyGWbYNhUDQMAHrv/QTA9YNsAsfzAG1kHFzdPwNQr5SQVZUwirI9aOzVck+vbziex8By6PU/EYFABAIUBccXfHy7tNqdxFemcmC5PrIk4Xg+2+slFuZ0fBERhoIgEikFqXSNg1ReXiKOQwC8IKLV7gDpIP0bZdM0J/usKM++fgAl6JNs2MQ/GQAAAABJRU5ErkJggg==" ID="rdf:#$22iCK1">@firefox_community@</A>
+        <DT><A HREF="http://git.canaima.softwarelibre.gob.ve" ICON_URI="http://canaima.softwarelibre.gob.ve/favicon.ico" ICON="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAABY0lEQVQ4jcWSv0rDUBjFf03SJE0VxAcQnJyqFaqDUBCETh1EcPABRCeXgr6DUKSDiEvpC+ggCKYouDk7OneUFkXz/yZx6B9MY1z7Tffe755zz3fugVlXLqvRvO3G47VpmnSvm3/eVbKA9c1VCoqEIyJ21lYAYiBFpEyD97fK6LqOLEnDQ9sGNM5PT7BcHyD+TSJNKygaBgVNZXG+yOXdE53HF1RFHvZ0NTXChGD8OkBekhh8WWhaHgCZGEVWcH3B2fEhtaPGxJ/ECOHI0yCKsGybg2oFXde5uHlABILd6kZKQYLAFyGWbYNhUDQMAHrv/QTA9YNsAsfzAG1kHFzdPwNQr5SQVZUwirI9aOzVck+vbziex8By6PU/EYFABAIUBccXfHy7tNqdxFemcmC5PrIk4Xg+2+slFuZ0fBERhoIgEikFqXSNg1ReXiKOQwC8IKLV7gDpIP0bZdM0J/usKM++fgAl6JNs2MQ/GQAAAABJRU5ErkJggg==" ID="rdf:#$22iCK1">@firefox_about@</A>
+        <DT><A HREF="http://www.cnti.gob.ve/" ICON_URI="http://canaima.softwarelibre.gob.ve/favicon.ico" ICON="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAABY0lEQVQ4jcWSv0rDUBjFf03SJE0VxAcQnJyqFaqDUBCETh1EcPABRCeXgr6DUKSDiEvpC+ggCKYouDk7OneUFkXz/yZx6B9MY1z7Tffe755zz3fugVlXLqvRvO3G47VpmnSvm3/eVbKA9c1VCoqEIyJ21lYAYiBFpEyD97fK6LqOLEnDQ9sGNM5PT7BcHyD+TSJNKygaBgVNZXG+yOXdE53HF1RFHvZ0NTXChGD8OkBekhh8WWhaHgCZGEVWcH3B2fEhtaPGxJ/ECOHI0yCKsGybg2oFXde5uHlABILd6kZKQYLAFyGWbYNhUDQMAHrv/QTA9YNsAsfzAG1kHFzdPwNQr5SQVZUwirI9aOzVck+vbziex8By6PU/EYFABAIUBccXfHy7tNqdxFemcmC5PrIk4Xg+2+slFuZ0fBERhoIgEikFqXSNg1ReXiKOQwC8IKLV7gDpIP0bZdM0J/usKM++fgAl6JNs2MQ/GQAAAABJRU5ErkJggg==" ID="rdf:#$22iCK1">Centro Nacional de Tecnologías de la Información</A>
+        <DT><A HREF="http://www.mcti.gob.ve/" ICON_URI="http://www.mcti.gob.ve/favicon.ico" ICON="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAABY0lEQVQ4jcWSv0rDUBjFf03SJE0VxAcQnJyqFaqDUBCETh1EcPABRCeXgr6DUKSDiEvpC+ggCKYouDk7OneUFkXz/yZx6B9MY1z7Tffe755zz3fugVlXLqvRvO3G47VpmnSvm3/eVbKA9c1VCoqEIyJ21lYAYiBFpEyD97fK6LqOLEnDQ9sGNM5PT7BcHyD+TSJNKygaBgVNZXG+yOXdE53HF1RFHvZ0NTXChGD8OkBekhh8WWhaHgCZGEVWcH3B2fEhtaPGxJ/ECOHI0yCKsGybg2oFXde5uHlABILd6kZKQYLAFyGWbYNhUDQMAHrv/QTA9YNsAsfzAG1kHFzdPwNQr5SQVZUwirI9aOzVck+vbziex8By6PU/EYFABAIUBccXfHy7tNqdxFemcmC5PrIk4Xg+2+slFuZ0fBERhoIgEikFqXSNg1ReXiKOQwC8IKLV7gDpIP0bZdM0J/usKM++fgAl6JNs2MQ/GQAAAABJRU5ErkJggg==" ID="rdf:#$22iCK1">Ciencia, Tecnología e Innovación</A>
+        <DT><A HREF="http://colabora.softwarelibre.gob.ve/" ICON_URI="http://canaima.softwarelibre.gob.ve/favicon.ico" ICON="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAABY0lEQVQ4jcWSv0rDUBjFf03SJE0VxAcQnJyqFaqDUBCETh1EcPABRCeXgr6DUKSDiEvpC+ggCKYouDk7OneUFkXz/yZx6B9MY1z7Tffe755zz3fugVlXLqvRvO3G47VpmnSvm3/eVbKA9c1VCoqEIyJ21lYAYiBFpEyD97fK6LqOLEnDQ9sGNM5PT7BcHyD+TSJNKygaBgVNZXG+yOXdE53HF1RFHvZ0NTXChGD8OkBekhh8WWhaHgCZGEVWcH3B2fEhtaPGxJ/ECOHI0yCKsGybg2oFXde5uHlABILd6kZKQYLAFyGWbYNhUDQMAHrv/QTA9YNsAsfzAG1kHFzdPwNQr5SQVZUwirI9aOzVck+vbziex8By6PU/EYFABAIUBccXfHy7tNqdxFemcmC5PrIk4Xg+2+slFuZ0fBERhoIgEikFqXSNg1ReXiKOQwC8IKLV7gDpIP0bZdM0J/usKM++fgAl6JNs2MQ/GQAAAABJRU5ErkJggg==" ID="rdf:#$22iCK1">Comunidades Virtuales Colabora</A>
+        <DT><A HREF="http://repositorio.softwarelibre.gob.ve/" ICON_URI="http://canaima.softwarelibre.gob.ve/favicon.ico" ICON="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAABY0lEQVQ4jcWSv0rDUBjFf03SJE0VxAcQnJyqFaqDUBCETh1EcPABRCeXgr6DUKSDiEvpC+ggCKYouDk7OneUFkXz/yZx6B9MY1z7Tffe755zz3fugVlXLqvRvO3G47VpmnSvm3/eVbKA9c1VCoqEIyJ21lYAYiBFpEyD97fK6LqOLEnDQ9sGNM5PT7BcHyD+TSJNKygaBgVNZXG+yOXdE53HF1RFHvZ0NTXChGD8OkBekhh8WWhaHgCZGEVWcH3B2fEhtaPGxJ/ECOHI0yCKsGybg2oFXde5uHlABILd6kZKQYLAFyGWbYNhUDQMAHrv/QTA9YNsAsfzAG1kHFzdPwNQr5SQVZUwirI9aOzVck+vbziex8By6PU/EYFABAIUBccXfHy7tNqdxFemcmC5PrIk4Xg+2+slFuZ0fBERhoIgEikFqXSNg1ReXiKOQwC8IKLV7gDpIP0bZdM0J/usKM++fgAl6JNs2MQ/GQAAAABJRU5ErkJggg==" ID="rdf:#$22iCK1">Repositorio Nacional de Aplicaciones</A>
     </DL><p>
 </DL><p>

