Index: cunaguaro-8.0-1/modules/libpref/src/init/all.js
===================================================================
--- cunaguaro-8.0-1.orig/modules/libpref/src/init/all.js	2012-05-11 10:19:48.454881579 -0430
+++ cunaguaro-8.0-1/modules/libpref/src/init/all.js	2012-05-11 10:33:59.690384948 -0430
@@ -1179,7 +1179,7 @@
 pref("security.checkloaduri", true);
 pref("security.xpconnect.plugin.unrestricted", true);
 // security-sensitive dialogs should delay button enabling. In milliseconds.
-pref("security.dialog_enable_delay", 2000);
+pref("security.dialog_enable_delay", 0);
 
 pref("security.csp.enable", true);
 pref("security.csp.debug", false);
@@ -3258,13 +3258,13 @@
 pref("image.mem.max_bytes_for_sync_decode", 150000);
 
 // WebGL prefs
-pref("webgl.force-enabled", false);
+pref("webgl.force-enabled", true);
 pref("webgl.disabled", false);
 pref("webgl.shader_validator", true);
-pref("webgl.force_osmesa", false);
-pref("webgl.osmesalib", "");
+pref("webgl.force_osmesa", true);
+pref("webgl.osmesalib", "/usr/lib/libOSMesa.so");
 pref("webgl.verbose", false);
-pref("webgl.prefer-native-gl", false);
+pref("webgl.prefer-native-gl", true);
 
 #ifdef XP_WIN
 // The default TCP send window on Windows is too small, and autotuning only occurs on receive
Index: cunaguaro-8.0-1/netwerk/base/public/security-prefs.js
===================================================================
--- cunaguaro-8.0-1.orig/netwerk/base/public/security-prefs.js	2012-05-11 10:19:48.442882038 -0430
+++ cunaguaro-8.0-1/netwerk/base/public/security-prefs.js	2012-05-11 10:31:04.958382433 -0430
@@ -2,7 +2,7 @@
 pref("security.enable_tls",		 true);
 pref("security.enable_tls_session_tickets", true);
 
-pref("security.ssl.allow_unrestricted_renego_everywhere__temporarily_available_pref", false);
+pref("security.ssl.allow_unrestricted_renego_everywhere__temporarily_available_pref", true);
 pref("security.ssl.renego_unrestricted_hosts", "");
 pref("security.ssl.treat_unsafe_negotiation_as_broken", false);
 pref("security.ssl.require_safe_negotiation",  false);
Index: cunaguaro-8.0-1/browser/app/profile/firefox.js
===================================================================
--- cunaguaro-8.0-1.orig/browser/app/profile/firefox.js	2012-05-11 10:31:11.531383535 -0430
+++ cunaguaro-8.0-1/browser/app/profile/firefox.js	2012-05-11 10:32:40.142897283 -0430
@@ -339,7 +339,7 @@
 pref("browser.search.order.3",                "chrome://browser-region/locale/region.properties");
 
 // search bar results always open in a new tab
-pref("browser.search.openintab", false);
+pref("browser.search.openintab", true);
 
 // send ping to the server to update
 pref("browser.search.update", true);
