Index: cunaguaro-6.0+0/configure
===================================================================
--- cunaguaro-6.0+0.orig/configure	2011-09-26 15:32:33.979636833 -0430
+++ cunaguaro-6.0+0/configure	2011-09-26 15:37:42.043135907 -0430
@@ -24599,7 +24599,7 @@
 # impacts profile location and user-visible fields.
 # - MOZ_APP_BASENAME: Typically stays consistent for multiple branded
 # versions of a given application (e.g. Aurora and Firefox both use
-# "Firefox"), but may vary for full rebrandings (e.g. Iceweasel). Used
+# "Firefox"), but may vary for full rebrandings (e.g. Cunaguaro). Used
 # for application.ini's "Name" field, which controls profile location in
 # the absence of a "Profile" field (see below), and various system
 # integration hooks (Unix remoting, Windows MessageWindow name, etc.)
Index: cunaguaro-6.0+0/configure.in
===================================================================
--- cunaguaro-6.0+0.orig/configure.in	2011-09-26 15:32:33.979636833 -0430
+++ cunaguaro-6.0+0/configure.in	2011-09-26 15:37:42.047136042 -0430
@@ -8814,7 +8814,7 @@
 # impacts profile location and user-visible fields.
 # - MOZ_APP_BASENAME: Typically stays consistent for multiple branded
 # versions of a given application (e.g. Aurora and Firefox both use
-# "Firefox"), but may vary for full rebrandings (e.g. Iceweasel). Used
+# "Firefox"), but may vary for full rebrandings (e.g. Cunaguaro). Used
 # for application.ini's "Name" field, which controls profile location in
 # the absence of a "Profile" field (see below), and various system
 # integration hooks (Unix remoting, Windows MessageWindow name, etc.)
Index: cunaguaro-6.0+0/testing/firebug/mozrunner/mozrunner/runner.py
===================================================================
--- cunaguaro-6.0+0.orig/testing/firebug/mozrunner/mozrunner/runner.py	2011-09-26 15:32:33.983637265 -0430
+++ cunaguaro-6.0+0/testing/firebug/mozrunner/mozrunner/runner.py	2011-09-26 15:37:42.047136042 -0430
@@ -228,7 +228,7 @@
     if sys.platform == 'darwin':
         names = ['firefox', 'minefield', 'shiretoko']
     elif (sys.platform == 'linux2') or (sys.platform in ('sunos5', 'solaris')):
-        names = ['firefox', 'mozilla-firefox', 'iceweasel']
+        names = ['firefox', 'mozilla-firefox', 'cunaguaro']
     elif os.name == 'nt' or sys.platform == 'cygwin':
         names =['firefox']
     else:
Index: cunaguaro-6.0+0/testing/mozmill/mozrunner/mozrunner/__init__.py
===================================================================
--- cunaguaro-6.0+0.orig/testing/mozmill/mozrunner/mozrunner/__init__.py	2011-09-26 15:32:33.983637265 -0430
+++ cunaguaro-6.0+0/testing/mozmill/mozrunner/mozrunner/__init__.py	2011-09-26 15:37:42.047136042 -0430
@@ -310,7 +310,7 @@
         if sys.platform == 'darwin':
             return ['firefox', 'minefield', 'shiretoko']
         if (sys.platform == 'linux2') or (sys.platform in ('sunos5', 'solaris')):
-            return ['firefox', 'mozilla-firefox', 'iceweasel']
+            return ['firefox', 'mozilla-firefox', 'cunaguaro']
         if os.name == 'nt' or sys.platform == 'cygwin':
             return ['firefox']
 
@@ -495,7 +495,7 @@
         if sys.platform == 'darwin':
             return ['firefox', 'minefield', 'shiretoko']
         if (sys.platform == 'linux2') or (sys.platform in ('sunos5', 'solaris')):
-            return ['firefox', 'mozilla-firefox', 'iceweasel']
+            return ['firefox', 'mozilla-firefox', 'cunaguaro']
         if os.name == 'nt' or sys.platform == 'cygwin':
             return ['firefox']
 
