From: Mike Hommey <mh@glandium.org>
Date: Fri, 5 Apr 2013 09:51:33 +0200
Subject: Don't install system profile

---
 xulrunner/app/moz.build |    2 --
 1 file changed, 2 deletions(-)

diff --git a/xulrunner/app/moz.build b/xulrunner/app/moz.build
index 5f33505..89251dc 100644
--- a/xulrunner/app/moz.build
+++ b/xulrunner/app/moz.build
@@ -2,5 +2,3 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-DIRS += ['profile']
